streamly.cabal

```diff
@@ -307,7 +307,9 @@ library
                      , Streamly.Internal.Data.Time.Clock
                      , Streamly.Internal.Data.SVar
                      , Streamly.Internal.Data.Array
+                     , Streamly.Internal.Data.Prim.Array.Types
                      , Streamly.Internal.Data.Prim.Array
+                     , Streamly.Internal.Data.SmallArray.Types
```

Comment:
No need to export for GHC versions that come with a primitive version that contains SmallArray

Reply:
That was the initial plan but We will not be using `SmallArray` from `primitive`.